internal/uri/uri_test.go

```diff
@@ -1,11 +1,64 @@
-package install
+package uri
 
 import (
-	"testing"
-
 	"github.com/stretchr/testify/assert"
+	"testing"
 )
 
+func TestExtractUrl(t *testing.T) {
+	type test struct {
+		inputURL    string
+		expectedURL string
+	}
+
+	tests := []test{
+		{"https://example.com/testdb.bak,myDbName", "https://example.com/testdb.bak"},
+		{"https://example.com/testdb.bak", "https://example.com/testdb.bak"},
+		{"https://example.com,", "https://example.com,"},
+	}
+
+	for _, testcase := range tests {
+		u := NewUri(testcase.inputURL)
+		assert.Equal(t, testcase.expectedURL, u.ActualUrl(),
+			"Extracted URL does not match expected URL")
+	}
+}
+
+func TestParseDbName(t *testing.T) {
+	type test struct {
+		inputURL    string
+		expectedURL string
+	}
+
+	tests := []test{
+		{"https://example.com/testdb.bak,myDbName", "myDbName"},
```

Comment:
>		{"https://example.com/testdb.bak,myDbName" [](http://example.com/codeflow?start=0&length=44)

It's a bit odd to conflate "URL + a database name" as "Uri"

The fact that `--use` flag and "use" command take this format as a parameter is specific to the semantic of those commands, correct? 
I would expect these strings to be deserialized into something more meaningful like
```go
type DatabaseFile struct {
  Source *url.URL
  DatabaseName string
}
```